channels.yaml

```diff
@@ -133,7 +133,15 @@ releases:
       etcd-arg:
         type: array
     agentArgs: *agentArgs-v1
-  - version: v1.22.4+k3s1
+  - version: v1.21.8-rc1+k3s1
+    minChannelServerVersion: v2.6.0-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v2
+      <<: *serverArgs-v1
+      etcd-arg:
+        type: array
+    agentArgs: *agentArgs-v1
+  - version: v1.22.5-rc1+k3s1
```

Comment:
I think we want a whole new section for 1.22.5 instead of overwriting 1.22.4 but @kinarashah would probably know better.